assets/js/components/surveys/CurrentSurvey.stories.js

```diff
@@ -20,36 +20,74 @@
+const Template = ( { setupRegistry, ...args } ) => (
+	<WithRegistrySetup func={ setupRegistry }>
+		<CurrentSurvey { ...args } />
+	</WithRegistrySetup>
+);
 
-export const CurrentSurveyStory = Template.bind( {} );
-CurrentSurveyStory.storyName = 'CurrentSurvey';
+export const SurveySingleQuestionStory = Template.bind( {} );
+SurveySingleQuestionStory.storyName = 'Single question';
+SurveySingleQuestionStory.args = {
+	setupRegistry: ( registry ) => {
+		fetchMock.post( /google-site-kit\/v1\/core\/user\/data\/survey-event/, { body: {}, status: 200 } );
+
+		registry.dispatch( CORE_USER ).receiveTriggerSurvey( fixtures.singleQuestionSurvey, { triggerID: 'storybookSurvey' } );
+	},
+};
+
+export const SurveyMultipleQuestionsStory = Template.bind( {} );
+SurveyMultipleQuestionsStory.storyName = 'Multiple questions';
+SurveyMultipleQuestionsStory.args = {
+	setupRegistry: ( registry ) => {
+		fetchMock.post( /google-site-kit\/v1\/core\/user\/data\/survey-event/, { body: {}, status: 200 } );
+
+		registry.dispatch( CORE_USER ).receiveTriggerSurvey( fixtures.multiQuestionSurvey, { triggerID: 'storybookSurvey' } );
+	},
+};
+
+export const SurveyNotAnsweredNoFollowUpStory = Template.bind( {} );
+SurveyNotAnsweredNoFollowUpStory.storyName = 'New survey (no follow-up CTA)';
+SurveyNotAnsweredNoFollowUpStory.args = {
+	setupRegistry: ( registry ) => {
+		fetchMock.post( /google-site-kit\/v1\/core\/user\/data\/survey-event/, { body: {}, status: 200 } );
+
+		registry.dispatch( CORE_USER ).receiveTriggerSurvey( fixtures.singleQuestionSurveyWithNoFollowUp, { triggerID: 'storybookSurvey' } );
+	},
+};
+
+export const SurveyAnsweredPositiveStory = Template.bind( {} );
+SurveyAnsweredPositiveStory.storyName = 'Completed';
+SurveyAnsweredPositiveStory.args = {
+	setupRegistry: ( registry ) => {
+		fetchMock.post( /google-site-kit\/v1\/core\/user\/data\/survey-event/, { body: {}, status: 200 } );
+
+		registry.dispatch( CORE_USER ).receiveTriggerSurvey( fixtures.singleQuestionSurvey, { triggerID: 'storybookSurvey' } );
+
+		registry.dispatch( CORE_FORMS ).setValues(
+			`survey-${ fixtures.singleQuestionSurvey.session.session_id }`,
+			{
+				answers: [
+					{ question_ordinal: 1, answer_ordinal: 5 },
+				],
+			}
+		);
+	},
+};
 
 export default {
-	title: 'Components/Surveys',
+	title: 'Components/Surveys/CurrentSurvey',
 	decorators: [
-		( Story ) => {
-			const triggerID = 'test-survey';
-			const survey = {
-				survey_payload: 'foo',
-				session: 'bar',
-			};
-			const setupRegistry = async ( registry ) => {
-				provideSiteInfo( registry );
-
-				registry.dispatch( CORE_USER ).receiveTriggerSurvey( survey, { triggerID } );
-				await registry.dispatch( CORE_USER ).triggerSurvey( triggerID, { ttl: 1 } );
-			};
-
-			return (
-				<WithRegistrySetup func={ setupRegistry }>
-					<Story />
-				</WithRegistrySetup>
-			);
-		},
+		( Story ) => (
+			<div className="googlesitekit-survey">
+				<Story />
+			</div>
```

Comment:
This wrapping div + class needs to be part of the output of the component now as this is actually missing when it renders in the app now. I think you originally had just question / completion components here which is why we moved this to the decorator. This needs to be added for both questions and completions.